### ISSUE TITLE
Set map center point via link

### ISSUE THREAD

**Issue by darkforcesjedi** [seikur0/PGO-mapscan-opt#218]
I already committed this to the webmod branch on my fork. It will allow you to link to:

http://yourpokesite:port?lat=XXX&lng=YYY

The mapscan.js file will read this during initMap() and set the center of the map accordingly. If the query string has not been set it continues to use the default origin.

Instead of using any of the built-in notification services I have a script that will check the database periodically and send me an email with a list of all the Pokemon on my alert list that have spawned with links that open the map to their coordinates.

**Comment by seikur0** [comment]
Would be so much easier with Flask :P I've been thinking, instead of doing an alternate server, why can't we just integrate ssl?

**Comment by darkforcesjedi** [comment]
You could add SSL, but a full featured web server offers so many possibilities you could never support them all. I am sure there are much better ways to implement it than I did.

**Comment by darkforcesjedi** [comment]
I reconfigured my web server to locally proxy localhost:port over SSL. I am going to delete my fork because realistically I won't be able to keep it up to date in parallel with your work. (And I would much rather play around with the new features than try to maintain the separate fork.)

Also, I browsed through the new code. I see the new version will save sessions to the database that can be resumed when main0 is restarted. Very useful. Is there any documentation forthcoming on the changes?

**Comment by seikur0** [comment]
Oh yeah I added session saving, that's falling under "many more small things that contribute to a smooth user experience" :D And this time around, I actually put some wiki pages up for the new changes.

**Comment by darkforcesjedi** [comment]
What are your intentions as far as what to implement next?

I was thinking about adding selectable layers to the map for Poké Stops, gyms, and spawn points (maybe a bad idea due to the number of points there are) as well as putting a Google maps link on Pokémon labels to make it easy to navigate to a Pokémon.

**Comment by seikur0** [comment]
I want to flesh out the spawnview tool a little bit more, maybe change the whole map interface to leaflet, if that's faster. By the way to add pokestops and gyms and spawn points, it's probably good to just draw them on the tiles and use custom tiles, that should be possible with leaflet. At least the heatmap feature should be separate from the main map, since it's not needed for mobile users. Then the integration of scanning lured pokestops/gyms. And Telegram enhancements. Not necessarily in that order. Oh yeah and authentification, something like an account system (easy to do with flask).
